runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java

```diff
@@ -96,4 +96,34 @@ default int getPrayerThreshold()
 	{
 		return 0;
 	}
+
+	@ConfigItem(
+		keyName = "oxygenChatAlert",
+		name = "Oxygen Chat Message",
+		position = 7,
+		description = "Get a text message warning you when your oxygen is about to run out")
+	default boolean oxygenChatAlert()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "oxygenNotifyAlert",
+		name = "Oxygen Notifier Message",
+		position = 8,
+		description = "Get a notifier even when not in runelite when your oxygen is about to run out")
+	default boolean oxygenNotifyAlert()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "oxygenThreshold",
+		name = "Oxygen Threshold",
```

Comment:
Follow naming convention of prayer/hp threshold etc. E.g, this should be
```
@ConfigItem(
  keyName = "oxygen",
  name = "Oxygen Notification Threshold",
  decription = "The amount of remaining oxygen to send a notification at. A value of 0 will disable notification."
)
default int getOxygenThreshold()
{
  return 0;
}
```

Remove rest of config for oxygen alerter except this